### PR TITLE
[Bugfix] Test_copp bugfix for backend testbeds

### PR DESCRIPTION
**Pull request by developfast**
### Description of PR

Summary: Currently, test_copp will fail on backend testbeds due to the addition on finding upstream DUTs introduced by #5097 which do not exist on T1 backend testbeds. Hence, I have added checks to ensure this type of lookup is skipped for backend testbeds.
Fixes # (issue)

### Type of change

- [X] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [X] 202205
- [X] 202305
- [X] 202311

#### What is the motivation for this PR?
Bug fix for backend testbeds
#### How did you do it?
Check if current testbed is backend and allow for skips for upstream DUTs.
#### How did you verify/test it?
Tested on lab backend device + frontend device

### CODE REVIEW

File: tests/copp/conftest.py [hunk 1]
```diff
@@ -27,3 +30,15 @@ def pytest_addoption(parser):
         default="cold",
         help="reboot type such as cold, fast, warm, soft"
     )
+
+
+@pytest.fixture(autouse=True, scope="module")
```
Comment (neethajohn) [inline tests/copp/conftest.py:35]: Changes look good. Please share the results from a non backend testbed as well

Comment (developfast) [reply tests/copp/conftest.py:35]: Passes on frontend T0 
`================================================ 10 passed, 1 warning in 1491.24s (0:24:51) ================================================`

**Review by neethajohn — APPROVED** [review]

**Comment by developfast** [comment]
@yxieca can you pls help merge?

**Comment by neethajohn** [comment]
@StormLiangMS , please help with cherry-pick to 202305

**Pull request merged**